docs/webiny-apps/headless-cms/features/environments.md

```diff
@@ -0,0 +1,26 @@
+---
+id: environments
+title: Environments
+sidebar_label: Environments
+---
+
+Environments are a powerful feature in Webiny Headless CMS.
```

Comment:
Here you have Environments in capitals.
In alias' you have environments in lowercase. I'd suggest either pick always put Environments, or use environments.
(personally I learn towards environment, unless it's a title or a start of the sentence)

Reply:
It's a start of the sentence.